pkg/apis/operator/v1alpha1/elevator_types.go

```diff
@@ -211,6 +219,61 @@ func (s *Elevator) SetUnReadyCondition() bool {
 	return false
 }
 
+// IsPaused returns if the Elevator is paused.
+func (s *Elevator) IsPaused() bool {
+	if s.Generation != s.Status.ObservedGeneration {
+		return false
+	}
+
+	for _, condition := range s.Status.Conditions {
+		if condition.Type == ElevatorPaused &&
+			condition.Status == ConditionTrue {
+			return true
+		}
+	}
+	return false
+}
+
+func (s *Elevator) SetPausedCondition() bool {
+	var changed bool
+	if !s.IsReady() {
+		changed = true
+		s.Status.ObservedGeneration = s.Generation
+		s.Status.SetCondition(ElevatorCondition{
+			Type:    ElevatorReady,
+			Status:  ConditionTrue,
+			Reason:  "Paused",
+			Message: "Reconcilation is paused, assuming component is ready.",
+		})
+		return changed
+	}
+	if !s.IsPaused() {
+		changed = true
+		s.Status.ObservedGeneration = s.Generation
+		s.Status.SetCondition(ElevatorCondition{
+			Type:    ElevatorPaused,
+			Status:  ConditionTrue,
+			Reason:  "Paused",
+			Message: "Reconcilation is paused",
+		})
+	}
+	return changed
+}
+
+func (s *Elevator) SetUnPausedCondition() bool {
```

Comment:
```suggestion
func (s *Elevator) Unpause() bool {
```

Reply:
let's leave it consistent with `SetUnReadyCondition`